Classes/Aspects/ThumbnailAspect.php

```diff
@@ -109,6 +109,15 @@ public function generateImgproxyUri(JoinPointInterface $joinPoint): ?array
 
         $expectedSize = ImgproxyBuilder::expectedSize($actualDimension, $targetDimension, $resizingType, $enlarge);
 
+        foreach ($this->settings['imgproxyUrlModifiers'] as $modifierClassName){
+            if(class_exists($modifierClassName)){
```

Comment:
IMHO `class_exists()` and `is_callable()` should be removed here. When class does not exist or isn't callable, then i prefer to get an Exception and to know i made a mistake.